pyvista/plotting/composite_mapper.py

```diff
@@ -0,0 +1,769 @@
+"""Module containing composite data mapper."""
+from itertools import cycle
+import logging
+import sys
+from typing import Optional
+import weakref
+
+import numpy as np
+
+from pyvista import _vtk
+from pyvista.utilities import convert_array, convert_string_array
+
+from ..utilities.misc import has_module, vtk_version_info
+from .colors import Color, get_cmap_safe
+
+
+class BlockAttributes:
+    """Block attributes used to set the attributes of a block.
+
+    Parameters
+    ----------
+    block : pyvista.DataObject
+        PyVista dataobject.
+
+    attr : pyvista.CompositeAttributes
```

Comment:
For some reason in my local build intersphinx is picking up on `pyvista.DataObject` but not `pyvista.CompositeAttributes`. Any idea why?